pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/loaders/UploadStatsLoader.kt

```diff
@@ -155,25 +155,14 @@ class UploadStatsLoader: KoinComponent {
         timeRangeWhereClause: String
     ): List<UndeliveredUpload> {
         try {
-            //Get the uploadIds for each upload with 'upload-completed' and no report for blob-file-copy
-            val undeliveredUploadIdsForBlobFileCopy = getUploadIdsWithIssues(
-                dataStreamId,
-                dataStreamRoute,
-                "upload-completed",
-                "blob-file-copy",
-                timeRangeWhereClause
-            )
 
+            //All the uploadIds that can be categorized as undelivered
             //Query - Get all the uploads with an item for blob-file-copy and status of failure
             val unDeliveredUploadIdsQuery = buildBlobFileCopyFailureQuery(dataStreamId, dataStreamRoute, timeRangeWhereClause)
```

Comment:
The AC for this is:
- Update the undeliveredUploads to be for a given upload ID, does one or more of the file-copy reports exist and do all of them have a success status, otherwise it is undelivered.

I see that this query will find the uploads that have a failure with the file copy, but what about the case where the file-copy report is completely missing?  That is also considered an undelivered upload.  This logic doesn't appear to catch that.

Reply:
Updated the code.